Projekty/Projekt1/PatrykTomaszewskiMateuszZiemla/Milestone 2.ipynb

```diff
@@ -0,0 +1,503 @@
+{
```

Comment:
<p>Chopaki, ja szanuję to, że umiecie kerasy i tensorfloły, ale trochę nie o tym jest ten przedmiot. </p><p> </p><p>Też nie samym acc człowiek żyje, warto to przedstawić w jakiś uporządkowany sposób. Czyli komentarze, base line, testowanie architektur, jakieś wykresiki wyników. </p><p> </p><p>Jesteśmy dorośli, ja Was nie będę oblewać za to, że nie robicie czegoś wedle mojego widzimisię, ale daję Wam znać, że Wasze podejście w "dorosłym życiu" może nie być optymalne</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/74/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>Zerknijcie tutaj dla porównania: https://github.com/mini-pw/2021L-WUM/blob/b3a1691665b0105ca079727102bd20b260efb153/Projekty/Projekt1/Piasecki_Sinski/Feature-engineering-baseline-model-2.pdf</p>